charts/operator/values.yaml

```
@@ -32,3 +32,6 @@ clusterRole:
     - apiGroups: ["*"]
       resources: ["*"]
       verbs: ["*"]
+
+airgapped: false
+charts: []
```

List can we do key string? and iterate object?
```suggestion
charts: {}
```